packages/core/package.json

```diff
@@ -8,6 +8,7 @@
     "@maticnetwork/fx-portal": "^1.0.4",
     "@openzeppelin/contracts": "4.4.2",
     "@uma/common": "^2.28.4",
+    "@uma/merkle-distributor": "^1.3.38",
```

Comment:
This is needed to address a CI issue present from the most recent removal of packages from the repo.